ci.hocon

```diff
@@ -423,8 +423,16 @@ test-gems: ${gem-test-pack} {
   ]
 }
 
-test-ecosystem: ${gem-test-pack} {
-  setup: ${common.setup} ${gem-test-pack.setup}
+test-bundle: ${sulong} {
+  setup: ${common.setup} ${sulong.setup}
+  
+  run: [
+    ${jt} [test, bundle]
```

Comment:
`jt test bundle` is already run with Sulong as part of `test-cexts`, maybe we want to keep testing the variant without Sulong here?